_release/cn/saga_releaseNotes.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Release Notes"
+lang: cn
+ref: release
+permalink: /cn/release/saga-release-notes/
+excerpt: "Release Notes"
+last_modified_at: 2018-03-28T00:50:43-55:00
+---
+
+        Release Notes - Apache ServiceComb - Version saga-0.1.0
```

Comment:
It's not a md file, we need to fix the format.

Reply:
but it JekyII theme is able to handle it

Reply:
Interesting，let me have a look.